ccm_web/client/src/pages/Home.tsx

```diff
@@ -57,7 +57,7 @@ function Home (props: HomeProps): JSX.Element {
     setCourseNameAsync,
     (course: CanvasCourseBase|undefined) => {
       props.setCourse(course)
-      enqueueSnackbar('Course name saved', { variant: 'success' })
+      enqueueSnackbar('Course name saved, please refresh page.', { variant: 'success' })
```

Comment:
Does this language seem clearer?
```suggestion
      enqueueSnackbar('Course name was saved! Please refresh page to see the change in Canvas.', { variant: 'success' })
```